nncf/openvino/quantization/helpers.py

```diff
@@ -125,8 +129,67 @@ def quantize_impl(model: ov.Model,
         }
     ]
 
-    pot_dataloader = POTDataLoader(calibration_dataset)
-    engine = OVEngine(engine_config, pot_dataloader, pot_dataloader)
+    engine = OVEngine(engine_config, calibration_dataset, calibration_dataset)
+    pipeline = pot.create_pipeline(algorithms, engine)
+    compressed_model = pipeline.run(pot_model)
+    pot.compress_model_weights(compressed_model)
+
+    quantized_model = _convert_compressed_model_to_openvino_model(compressed_model)
+
+    return quantized_model
+
+
+def quantize_with_accuracy_control_impl(model: ov.Model,
+                                        calibration_dataset: Dataset,
+                                        validation_dataset: Dataset,
+                                        validation_fn: Callable[[ov.Model, Iterable[Any]], float],
+                                        max_drop: float = 0.01,
+                                        preset: QuantizationPreset = QuantizationPreset.PERFORMANCE,
+                                        target_device: TargetDevice = TargetDevice.ANY,
+                                        subset_size: int = 300,
+                                        fast_error_correction: bool = True,
+                                        model_type: Optional[str] = None,
+                                        ignored_scope: Optional[IgnoredScope] = None) -> ov.Model:
```

Comment:
Where are the other AccuracyAware parameters? Such as `ranking_subset_size`, `max_iter_num`, `drop_type`, `tune_hyperparams` and etc. Without these parameters, we cannot claim that we currently have achieved parity between NNCF & POT.
@AlexKoff88, what do you think?

Reply:
These are advanced parameters that should be introduced in the additional class in the follow up PRs.

Reply:
I would recommend to handle `drop_type` in this PR because it is very related to the `max_drop` parameter.

Reply:
I added the `DropType` param. Could you please look at this?